src/css/_utilities.scss

```diff
@@ -24,7 +24,6 @@
   border: 0;
   clip: rect(0 0 0 0);
   height: 1px;
-  margin: -1px;
```

Comment:
I'll copy/paste the commit description here:

> This looks like it was copy/pasted from HTML5 Boilerplate, but there
does not seem to be a purpose to the `margin: -1px;` line and it
causes unsightly "bumps" in the outline of certain elements (sliders).
>
> I browsed through 5 years of history of HTML5 Boilerplate and gave up
looking for a justification for this.
>
> The WebAIM link they even used to cite (and the snook.ca link they
currently cite) does not mention having this margin:
>
> http://webaim.org/techniques/css/invisiblecontent/